moveit_collisions_updater/README.md

```diff
@@ -0,0 +1,5 @@
+# MoveIt Collisions Updater
+
+To run the collision updater:
+
+ros2 run moveit_collisions_updater moveit_collisions_updater --urdf robot.urdf --srdf robot.srdf  --trials 100000
```

Comment:
You should also be able to set xacro args.
`ros2 run moveit_collisions_updater moveit_collisions_updater --urdf <path_to_urdf/xacro> --xacro-args <optional_xacro_args> --srdf <path_to_srdf> --trials 100000`